### PR TITLE
Make focusable elements inert when nav is offscreen

### PR DESCRIPTION
**Pull request by robdodson**

### CODE REVIEW

**Comment by paullewis** [comment]
I shall test this on the 'morrow, good sir. I'll be testing on a 5X, but it'd be interesting to extrapolate for older devices.

File: side-nav/index.html [hunk 1]
```diff
@@ -28,18 +28,25 @@
...
+          <li>About</li>
+          <li>Shop</li>
+          <li>Contact</li>
+        </ul>
+      </div>
     </nav>
   </aside>
-  <script src='./side-nav.js'></script>
+
+  <script src='./side-nav-es5.js'></script>
```
Comment (oldergod) [inline side-nav/index.html]: Could it have a been a local file you did not intend to commit?

Comment (robdodson) [inline side-nav/index.html]: ack yeah sorry let me fix that

**Comment by oldergod** [comment]
Tested on Nexus5 and results were good

slideIn
image

slideOut
image

**Comment by paullewis** [comment]
Closing because I prefer the detabinator approach ... for now.

**Pull request closed without merging**